Explain what white space is more pragmatically (#35818)
mdn/content: files/en-us/learn/css/first_steps/how_css_is_structured/index.md

## White space

> White space means actual spaces, tabs and new lines. Just as browsers ignore extra white space in HTML, browsers ignore extra white space inside CSS. The value of extra white space is how it can improve readability.

Reviewer: ```suggestion
White space means actual spaces, tabs and new lines. Just as browsers ignore extra white space in HTML, browsers ignore extra white space inside CSS. The value of white space is how it can improve readability.
```

> White space means actual spaces, tabs and new lines. Just as browsers ignore extra white space in HTML, browsers ignore extra white space inside CSS. The value of white space is how it can improve readability.

Reviewer: ```suggestion
White space means actual spaces, tabs and new lines. Just as browsers ignore extra white space in HTML, browsers ignore extra white space inside CSS. The benefit of white space is that it improves readability.
```

}
```

> The next example shows the equivalent CSS in a more compressed format without any extra white space. Although the two examples work the same, the one below is more difficult to read.

Reviewer: ```suggestion
The next example shows the equivalent CSS in a more compressed format, with all extra white space removed. Although the two examples work the same, the one below is more difficult to read.
```
